package.json

```diff
@@ -67,6 +68,7 @@
   "devDependencies": {
     "@changesets/cli": "^2.27.1",
     "@playwright/test": "^1.41.1",
+    "@types/global-agent": "^2.1.3",
```

Comment:
different version than global-agent?

Reply:
the latest release of @types/gloabal-agent is 2.1.3
(base) Mohds-MacBook-Pro:create-llama-amir amir$ pnpm view @types/global-agent versions
[ '2.1.0', '2.1.1', '2.1.2', '2.1.3' ]